Makefile

```diff
@@ -36,7 +36,7 @@ test: fmt vet ut
 # The target architecture is select by setting the ARCH variable.
 # When ARCH is undefined it is set to the detected host architecture.
 # When ARCH differs from the host architecture a crossbuild will be performed.
-ARCHES ?= $(patsubst build/Dockerfile.%,%,$(wildcard build/Dockerfile.*))
+ARCHES ?= amd64 arm64 ppc64le s390x
```

Comment:
We have a single Dockerfile now so the supported architectures are explicitly listed.